app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -45,6 +45,6 @@ public function store(Request $request): RedirectResponse
 
         Auth::login($user);
 
-        return redirect(route('index', absolute: false));
+        return redirect(route('employee', absolute: false));
```

Comment:
please revert this to dashboard. we have implemented a dashboard for common purposes not only for employee

Reply:
Reverted all changes done to Dashboard... create a new issue to properly implement the dashboard